optimism/src/mips/interpreter.rs

```diff
@@ -949,7 +949,19 @@ pub fn interpret_itype<Env: InterpreterEnv>(env: &mut Env, instr: ITypeInstructi
             return;
         }
         ITypeInstruction::SetLessThanImmediate => (),
-        ITypeInstruction::SetLessThanImmediateUnsigned => (),
+        ITypeInstruction::SetLessThanImmediateUnsigned => {
```

Comment:
Another unsigned in the name which works with signed value!! 
![image](https://github.com/o1-labs/proof-systems/assets/6018454/b59f275d-61a5-4ca0-b512-c27f49ad2e69)

Reply:
?

Reply:
Agree for the offset. It might be only me who thought that having `Unsigned` in the name would make clear that all computations, even the offset, would be unsigned :').